tensorboard/plugins/histogram/summary_test.py

```diff
@@ -119,6 +119,16 @@ def test_when_bucket_count_not_statically_known(self):
         buckets = tensor_util.make_ndarray(pb.value[0].tensor)
         self.assertEqual(buckets.shape, (bucket_count, 3))
 
+    def test_with_large_counts(self):
+        # Check for overflow with large count (2^24) of data.
+        large_count = 2 ** 24
```

Comment:
It's not actually enough to test 2^24, since that's the highest number that still works - we need to test a number greater than 2^24 (e.g. 2^24 + 1) in order to get the failure case.

Reply:
Done.